internal/backend/local/local.go

```diff
@@ -213,10 +213,10 @@ func (b *Local) Test(ctx context.Context, h restic.Handle) (bool, error) {
 	debug.Log("Test %v", h)
 	_, err := fs.Stat(b.Filename(h))
 	if err != nil {
-		if os.IsNotExist(errors.Cause(err)) {
+		if errors.Is(err, os.ErrNotExist) {
```

Comment:
Shouldn't that also use `IsNotExist`?

Reply:
The os.IsNotExist doc says "New code should use errors.Is(err, os.ErrNotExist)".

Reply:
More practically, if we use errors.Is everywhere, we can use custom errors on Windows that contain the fixed path as well as the original path, if they differ.

Reply:
Oh, I meant to write `b.IsNotExist`, which is also used in a few other places. Sorry for the confusion.

Reply:
Excuse me, I misunderstood. That is cleaner, changed the code.